### PR TITLE
feat(dynamicWidgets): release as production

### PR DESCRIPTION
**Pull request by Haroenv**
see FX-92

### CODE REVIEW

File: src/components/ExperimentalDynamicWidgets.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import AisDynamicWidgets from './DynamicWidgets';
+import { warn } from '../util/warn';
+
+// @MAJOR remove this file
```
Comment (dhayab) [inline src/components/ExperimentalDynamicWidgets.js:4]: Out of curiosity, how is this comment made actionable before a release? Is it by checking manually the codebase, or is there an automated check somewhere?

**Review by dhayab — APPROVED** [review]

File: src/components/ExperimentalDynamicWidgets.js (hunk 1 above)
Comment (Haroenv) [reply src/components/ExperimentalDynamicWidgets.js:4]: we check manually for now

File: src/components/ExperimentalDynamicWidgets.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+export default Object.assign({}, AisDynamicWidgets, {
```
Comment (sarahdayan) [inline src/components/ExperimentalDynamicWidgets.js:5]: Why `Object.assign` vs. spread?

File: src/components/ExperimentalDynamicWidgets.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  name: 'AisExperimentalDynamicWidgets',
+  mounted() {
+    warn('use AisDynamicWidgets instead of AisExperimentalDynamicWidgets');
```
Comment (sarahdayan) [inline src/components/ExperimentalDynamicWidgets.js:8]:
```suggestion
    warn('Use AisDynamicWidgets instead of AisExperimentalDynamicWidgets.');
```

**Review by sarahdayan — APPROVED** [review]

File: src/components/ExperimentalDynamicWidgets.js (hunk 2 above)
Comment (Haroenv) [reply src/components/ExperimentalDynamicWidgets.js:5]: Vue InstantSearch babel wasn't set up to have spread

**Pull request merged**